utils/frame/remote-externalities/src/lib.rs

```diff
@@ -428,10 +431,10 @@ impl<B: BlockT + DeserializeOwned> Builder<B> {
 		let keys_chunked: Vec<Vec<StorageKey>> =
 			keys.chunks(thread_chunk_size).map(|s| s.into()).collect::<Vec<_>>();
 		for thread_keys in keys_chunked {
-			let uri = Arc::clone(&uri);
+			let thread_client = client.clone();
 			let handle = std::thread::spawn(move || {
-				use async_std::task::block_on;
-				let thread_client = block_on(Transport::build_ws_client(&*uri)).unwrap();
+				let rt = tokio::runtime::Runtime::new().unwrap();
```

Comment:
have you actually tested this in `try-runtime-cli`?

Reply:
tested it right now by `RUST_LOG=remote-ext=debug,try-runtime-cli=debug ./target/release/substrate try-runtime offchain-worker live --uri wss://rpc.polkadot.io:443 --at 0x76b3e40351c4ea941f57233d0476259cfd21df418be01d456c05c55694c96f36 -p Proxy`

but I 'm just to lazy to run it on all pallets ^^